recipe/meta.yaml

```diff
@@ -39,21 +38,19 @@ requirements:
     - m2-base                      # [win]
   host:
     - python
-    - conda-package-handling >=1.3.0
-    - menuinst >=1.4.11,<2         # [win]
-    - packaging
     - pip
+    - hatchling >=1.12.2
+    - hatch-vcs >=0.2.0
+    - wheel
```

Comment:
@skupr-anaconda why did `wheel` get added here?

Reply:
oh, I see that this is one of the linter warnings:

> ERROR: /tmp/abs_00jpd09uts/clone/recipe/meta.yaml:47: missing_wheel: For pypi packages, wheel should be present in the host section

Reply:
Because conda-build implicitly adds wheel to `host` and then does, for example, this for linux-64:

```
  Building wheel for conda (pyproject.toml): started

  Building wheel for conda (pyproject.toml): finished with status 'done'
  Created wheel for conda: filename=conda-23.5.0-py3-none-any.whl size=730909 sha256=a8de6fa84d9f8abe3a412b8dd151bacac1b927b4125443a7d05826d548d2246f
```